### DISCUSSION TITLE
HmIP-SMI55-2 Tasten ohne Funktion

### DISCUSSION THREAD

**Discussion by stab2501** [OpenCCU/OpenCCU#2847] [🇩🇪 Fragen&Antworten] [answered]
Schönen guten Tag zusammen,

ich habe mir gestern ein paar HmIP-SMI55-2 gegönnt und dachte ich könnte dadurch die EnOcean und Aqara Geräte ersetzen.
Nun habe ich jedoch folgendes Problem nachdem ich die Bewegungsmelder in der RaspberryMatic angemeldet habe werden mir die Signale der Taster nirgends angezeigt. 
Die Funktion Bewegungsmelder funktioniert tadellos. 
In den Einstellungen finde ich auch Einstellmöglichkeiten für die Taster jedoch wird mir weder in der RaspberryMatic noch im HomeAssistant etwas angezeigt wenn ich die Taster drücke oder gedrückt halte. 

Im Netz konnte ich nichts zu einem Bug oder der gleichen finden...

Mache ich vielleicht irgendetwas falsch?

VG Freddy

**Comment by Baxxy13** [comment, marked as answer]
Es ist normal das die Zentrale "per default" keine Events von HmIP-Tastern empfängt.
Das lässt sich ändern:
https://github.com/SukramJ/custom_homematic/blob/devel/README.md#events-for-homematicip-devices

**Reply by stab2501** [reply]
Hey Baxxy13 du bist mein Held.

Vielen lieben Dank für die schnelle Hilfe.
